### PR TITLE
Cybersixgill xsoar alerts typosquatting

### PR DESCRIPTION
**Pull request by syed-loginsoft**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] Ready

## Related Issues
fixes: link to the issue

## Description
Added a command to populate context data for certain types of alerts.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by sapirshuker — CHANGES_REQUESTED** [review]
Hey, @syed-loginsoft  thanks a lot for this contribution, It is highly appreciated.
I have reviewed it but unfortunately it has some issues.
I have posted some comments in the review.
In addition, there are a build failures, please [review](https://app.circleci.com/jobs/github/demisto/content/723287) and fix them.
I'm available for any of your questions here or on slack.

File: Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.yml [hunk 1]
```diff
@@ -109,7 +109,116 @@ script:
...
+    name: cybersixgill-context
+    arguments:
+    - required: true
+      description: The alert id to fetch context.
+      name: alert_id
+    outputs:
+    - contextPath: Cybersixgill.Actor
+      description: The indicator that was tested.
+      type: string
+    - contextPath: Cybersixgill.AdditionalKeywords
```
Comment (sapirshuker) [inline Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.yml:122]: Context outputs should be in the form Cybersixgill.context.Actor etc..

**Comment by sapirshuker** [comment]
Hi @syed-loginsoft , we haven’t heard from you in a while.
Do you need any help with the pull request?

Please feel free to reach out to me here or on Slack.
Thanks again for contributing to our repo, hope to hear from you soon.

**Comment by syed-loginsoft** [comment]
@sapirshuker , Release notes already upto date. Guthub action is failing.
Please check

**Comment by sapirshuker** [comment]
Hey @syed-loginsoft , 
You accidentally deleted the Packs/Cybersixgill-ActionableAlerts/ReleaseNotes/1_2_6.md
and updated the Packs/Cybersixgill-ActionableAlerts/ReleaseNotes/1_2_5.md file.
Please revert those changes.
The current version of the pack is 1.2.6. It needs to be raised.
in order to do that after you revert the 1_2_6.md and the 1_2_5.md changes, please run the `demisto-sdk update-release-notes` command - for more information please see: https://xsoar.pan.dev/docs/concepts/demisto-sdk#update-release-notes.
I'm available for any of your questions here or on slack.

File: Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.yml (hunk 1 above)
Comment (sapirshuker) [reply Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.yml:130]: @syed-loginsoft Please note I added another comment.

**Comment by syed-loginsoft** [comment]
If I restore file and run command. It gets deleted.
image

**Comment by sapirshuker** [comment]
> If I restore file and run command. It gets deleted. image

@syed-loginsoft , you should merge from master and then retry.

**Comment by syed-loginsoft** [comment]
> > If I restore file and run command. It gets deleted. image
> 
> @syed-loginsoft , please try to merge from master and retry.

Done.

**Comment by kgal-pan** [comment]
Hi @syed-loginsoft  - As discussed in the email thread with the rest of the CyberSixGill team, we will be closing this PR as the introduced changes are not best practice and the solution should be redesigned before it can be contributed.

Thanks for the effort you put into this work!

**Pull request closed without merging**